.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,7 +33,7 @@ Please check the type of change your PR introduces:
 
 List of Issues closed or resolved by this PR. Add multiple `Closes` keyword followed by the issue number (e.g. Closes #ISSUE-NUMBER)
 
-* Closes #ISSUE-NUMBER
+Closes #ISSUE-NUMBER
```

Comment:
This works w/out `:` correct?

Reply:
That's correct. Confirmed through the previous PR to ensure the correct behavior